lib/features/home/view/widgets/property_item_view.dart

```diff
@@ -0,0 +1,376 @@
+                        child: Row(
+                          children: [
+                            CircleAvatar(
+                              radius: HsDimens.spacing20,
+                              backgroundImage: NetworkImage(widget.ownerAvatar),
+                            ),
+                            const SizedBox(
+                              width: HsDimens.spacing4,
+                            ),
+                            Expanded(
+                              child: Text(
+                                widget.ownerName,
+                                maxLines: 2,
+                                overflow: TextOverflow.ellipsis,
+                                style: Theme.of(context)
+                                    .textTheme
+                                    .bodySmall
+                                    ?.copyWith(
+                                        color: HSColor.neutral6,
+                                        fontWeight: FontWeight.w500),
+                              ),
+                            ),
+                            const SizedBox(width: 12),
+                            Expanded(
+                                child: RichText(
+                                    textAlign: TextAlign.end,
+                                    maxLines: 4,
+                                    overflow: TextOverflow.ellipsis,
+                                    text: TextSpan(
+                                        text:
+                                            '${HatSpaceStrings.current.availableDate}: ',
+                                        style: Theme.of(context)
+                                            .textTheme
+                                            .bodySmall
+                                            ?.copyWith(color: HSColor.neutral6),
+                                        children: [
+                                          TextSpan(
+                                            text: widget.availableDate,
+                                            style: Theme.of(context)
+                                                .textTheme
+                                                .bodySmall
+                                                ?.copyWith(
+                                                  color: HSColor.neutral8,
+                                                ),
+                                          )
+                                        ]))),
+                          ],
+                        ),
+                      ),
+                    ],
+                  ),
+                )),
+            Positioned(
+                top: HsDimens.spacing8,
+                right: HsDimens.spacing8,
+                child: ValueListenableBuilder<bool>(
+                  valueListenable: _favorited,
+                  builder: (_, favorited, __) {
+                    return InkWell(
```

Comment:
Use GestureDetector here, Inkwell comes with ripple effect which is not suitable for this icon.

Or you can use InkWel, but check with designer which ripple effect is good for this icon: square or circular